dev/com.ibm.websphere.appserver.features/visibility/public/mpHealth-3.0/io.openliberty.mpHealth-3.0.feature

```diff
@@ -0,0 +1,28 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpHealth-3.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-API-Package: \
+  org.eclipse.microprofile.health;  type="stable", \
+  org.eclipse.microprofile.health.spi;  type="stable"
+IBM-ShortName: mpHealth-3.0
+Subsystem-Name: MicroProfile Health 3.0
+-features=io.openliberty.org.eclipse.microprofile.health-3.0, \
+ com.ibm.websphere.appserver.cdi-2.0,\
+ com.ibm.websphere.appserver.classloaderContext-1.0, \
+ com.ibm.websphere.appserver.contextService-1.0, \
+ com.ibm.websphere.appserver.jndi-1.0, \
+ com.ibm.websphere.appserver.json-1.0, \
+ com.ibm.websphere.appserver.servlet-4.0,\
+ com.ibm.websphere.appserver.mpConfig-1.4; ibm.tolerates:=1.3,\
```

Comment:
MicroProfile 4.0 features should only depend on other 4.0 features ... in this case, please change this dependency to mpConfig-2.0

Reply:
Updated to use mpConfig-2.0.